.circleci/config.yml

```diff
@@ -121,7 +121,8 @@ commands:
       - run:
           name: checkout regression repo
           command: |
-            git clone https://github.com/vertexproject/synapse-regression ~/git/synapse-regression
+            # FIXME: Remove branch before merging
```

Comment:
Don't you forget about me
Don't, don't, don't, don't
Don't you forget about me